### PR TITLE
feat: [PIE-4627]: Wrap src with service for pipeline service

### PR DESCRIPTION
**Pull request by prashantsharma04**
## Checklist
- [x] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/35764)

### CODE REVIEW

**Comment by prashantsharma04** [comment]
trigger feature-build

**Comment by prashantsharma04** [comment]
trigger feature-build

**Comment by prashantsharma04** [comment]
trigger feature-build

**Comment by prashantsharma04** [comment]
trigger feature-build

**Comment by prashantsharma04** [comment]
trigger feature-build

File: .aeriform/BUILD.bazel [hunk 1]
```diff
@@ -178,9 +178,9 @@ aeriform("//490-ce-commons:module")
 
 aeriform("//490-ce-commons:tests")
 
-aeriform("//pipeline-service:module")
+aeriform("//pipeline-service/service:module")
 
-aeriform("//pipeline-service:tests")
```
Comment (prashantsharma04) [inline .aeriform/BUILD.bazel:183]: remove this

File: pipeline-service/build/build_jar.sh [hunk 2]
```diff
@@ -13,7 +13,7 @@ fi
 
 BAZEL_DIRS=${HOME}/.bazel-dirs
 BAZEL_ARGUMENTS="--show_timestamps --announce_rc --experimental_convenience_symlinks=normal --symlink_prefix=${BAZEL_DIRS}/"
-bazel ${bazelrc} build //pipeline-service:module_deploy.jar ${BAZEL_ARGUMENTS}
+bazel ${bazelrc} build //pipeline-service/service:module_deploy.jar ${BAZEL_ARGUMENTS}
```
Comment (prashantsharma04) [inline pipeline-service/build/build_jar.sh:16]: compare this with platform service build_jar.sh

File: pipeline-service/config/BUILD.bazel [hunk 3]
```diff
@@ -0,0 +1,9 @@
+exports_files(
+    [
+        "keystore.jks",
+        "config.yml",
+        #        "cert.pem",
+        #        "key.pem",
```
Comment (prashantsharma04) [inline pipeline-service/config/BUILD.bazel:6]: remove comment

**Comment by prashantsharma04** [comment]
trigger ti4

**Review by archit-harness — APPROVED** [review]

**Review by sahilhindwani-harness — APPROVED** [review]

**Pull request merged**